org.hl7.fhir.r4/src/test/java/org/hl7/fhir/r4/terminologies/ConceptMapEngineTest.java

```diff
@@ -0,0 +1,60 @@
+package org.hl7.fhir.r4.terminologies;
+
+import org.hl7.fhir.r4.context.SimpleWorkerContext;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.ConceptMap;
+import org.hl7.fhir.r4.model.Enumerations;
+import org.jetbrains.annotations.NotNull;
```

Comment:
For compatibility with hapi-fhir, we please use javax.annotation.Nonnull

It doesn't matter as much in test cases, but it will help prevent jetbrains uses from proliferating.

Reply:
I did a slight refactor here: https://github.com/hapifhir/org.hl7.fhir.core/tree/bug_check_loop_variable

I'm happy to make a PR of that after the Nonnull change is made, or you can merge it into yours.

Reply:
Just merged your branch into here